### ISSUE TITLE
Unable to insert content at URI: /ext/corb/process.xqy; cause: Need privilege

### ISSUE THREAD

**Issue by sammefford** [sastafford/marklogic-spring-batch#52]
I got the following error when I attempted 'gradle mlDeploy' with gradle.properties mlRestAdminUsername=rest-admin

I was able to resolve this by using the admin user.
```
13:41:59.925 [INFO] [com.sun.jersey.spi.service.ServiceFinder] Running in a non-OSGi environment
13:41:59.941 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface com.sun.jersey.client.proxy.ViewProxyProvider
13:41:59.941 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface com.sun.jersey.spi.inject.InjectableProvider
13:41:59.972 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.multipart.impl.MultiPartConfigProvider
13:41:59.972 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.SAXParserContextProvider
13:41:59.972 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.XMLStreamReaderContextProvider
13:41:59.972 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.DocumentBuilderFactoryProvider
13:41:59.972 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.TransformerFactoryProvider
13:42:00.019 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface javax.ws.rs.ext.MessageBodyReader
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.multipart.impl.MultiPartReaderClientSide
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.StringProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ByteArrayProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FileProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.InputStreamProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DataSourceProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.RenderedImageProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormMultivaluedMapProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$App
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$Text
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$General
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$App
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$Text
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$General
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$App
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$Text
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$General
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ReaderProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DocumentProvider
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$StreamSourceReader
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$SAXSourceReader
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$DOMSourceReader
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootObjectProvider$App
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootObjectProvider$Text
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootObjectProvider$General
13:42:00.066 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.EntityHolderReader
13:42:00.144 [INFO] [com.sun.jersey.core.spi.component.ProviderFactory] A dependent class, javax/mail/MessagingException, of the component class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider is not found. The component is ignored.
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface javax.ws.rs.ext.MessageBodyWriter
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.multipart.impl.MultiPartWriter
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.StringProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ByteArrayProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FileProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.InputStreamProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DataSourceProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.RenderedImageProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormMultivaluedMapProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$App
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$Text
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$General
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$App
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$Text
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$General
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$App
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$Text
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$General
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ReaderProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DocumentProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.StreamingOutputProvider
13:42:00.253 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$SourceWriter
13:42:00.269 [INFO] [com.sun.jersey.core.spi.component.ProviderFactory] A dependent class, javax/mail/MessagingException, of the component class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider is not found. The component is ignored.
13:42:00.316 [INFO] [com.marklogic.appdeployer.command.schemas.LoadSchemasCommand] Loading schemas database data from path: src/main/ml-schemas
13:42:00.347 [INFO] [com.marklogic.client.impl.DatabaseClientImpl] Releasing connection
13:42:00.347 [DEBUG] [com.marklogic.client.impl.JerseyServices] Releasing connection
13:42:00.347 [INFO] [com.marklogic.appdeployer.impl.SimpleAppDeployer] Finished executing command [com.marklogic.appdeployer.command.schemas.LoadSchemasCommand]

13:42:00.347 [INFO] [com.marklogic.appdeployer.impl.SimpleAppDeployer] Executing command [com.marklogic.appdeployer.command.modules.LoadModulesCommand] with sort order [400]
13:42:00.347 [DEBUG] [com.marklogic.client.DatabaseClientFactory] Creating new database client for server at localhost:8200
13:42:00.347 [DEBUG] [com.marklogic.client.impl.JerseyServices] Connecting to localhost at 8200 as rest-admin
13:42:00.347 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface com.sun.jersey.client.proxy.ViewProxyProvider
13:42:00.347 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface com.sun.jersey.spi.inject.InjectableProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.multipart.impl.MultiPartConfigProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.SAXParserContextProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.XMLStreamReaderContextProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.DocumentBuilderFactoryProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.xml.TransformerFactoryProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface javax.ws.rs.ext.MessageBodyReader
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.multipart.impl.MultiPartReaderClientSide
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.StringProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ByteArrayProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FileProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.InputStreamProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DataSourceProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.RenderedImageProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormMultivaluedMapProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$App
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$Text
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$General
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$App
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$Text
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$General
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$App
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$Text
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$General
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ReaderProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DocumentProvider
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$StreamSourceReader
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$SAXSourceReader
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$DOMSourceReader
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootObjectProvider$App
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootObjectProvider$Text
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootObjectProvider$General
13:42:00.363 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.EntityHolderReader
13:42:00.441 [DEBUG] [org.apache.http.impl.conn.tsccm.ThreadSafeClientConnManager] Shutting down
13:42:00.441 [INFO] [com.marklogic.client.impl.DatabaseClientImpl] Releasing connection
13:42:00.456 [INFO] [com.sun.jersey.core.spi.component.ProviderFactory] A dependent class, javax/mail/MessagingException, of the component class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider is not found. The component is ignored.
13:42:00.535 [INFO] [com.sun.jersey.core.spi.component.ProviderServices] Searching for providers that implement: interface javax.ws.rs.ext.MessageBodyWriter
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.multipart.impl.MultiPartWriter
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.StringProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ByteArrayProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FileProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.InputStreamProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DataSourceProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.RenderedImageProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.FormMultivaluedMapProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$App
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$Text
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLRootElementProvider$General
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$App
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$Text
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLJAXBElementProvider$General
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$App
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$Text
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.XMLListElementProvider$General
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.ReaderProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.DocumentProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.StreamingOutputProvider
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderServices]     Provider found: class com.sun.jersey.core.impl.provider.entity.SourceProvider$SourceWriter
13:42:00.550 [INFO] [com.sun.jersey.core.spi.component.ProviderFactory] A dependent class, javax/mail/MessagingException, of the component class com.sun.jersey.core.impl.provider.entity.MimeMultipartProvider is not found. The component is ignored.
13:42:00.566 [INFO] [com.marklogic.appdeployer.command.modules.LoadModulesCommand] Loading asset modules from dir: src/main/ml-modules
13:42:00.566 [DEBUG] [com.marklogic.client.modulesloader.impl.DefaultModulesLoader] Loading modules from base directory: E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\src\main\ml-modules
13:42:00.566 [DEBUG] [com.marklogic.client.modulesloader.impl.XccAssetLoader] Initializing XCC session; host: localhost; username: rest-admin; database name: spring-batch-modules
13:42:00.581 [DEBUG] [com.marklogic.xcc.spi.ConnectionProvider] constructing new SocketPoolProvider
13:42:00.597 [DEBUG] [com.marklogic.client.modulesloader.impl.XccAssetLoader] Loading assets from path: E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\src\main\ml-modules\ext
13:42:00.613 [DEBUG] [com.marklogic.client.modulesloader.impl.XccAssetLoader] Visiting directory: E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\src\main\ml-modules\ext
13:42:00.613 [DEBUG] [com.marklogic.client.modulesloader.impl.XccAssetLoader] Visiting directory: E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\src\main\ml-modules\ext\corb
13:42:00.628 [INFO] [com.marklogic.client.modulesloader.impl.XccAssetLoader] Inserting module with URI: /ext/corb/process.xqy
13:42:00.800 [DEBUG] [com.marklogic.client.modulesloader.impl.XccAssetLoader] Closing XCC session
13:42:00.800 [INFO] [com.marklogic.client.impl.DatabaseClientImpl] Releasing connection
13:42:00.800 [DEBUG] [com.marklogic.client.impl.JerseyServices] Releasing connection
13:42:00.800 [DEBUG] [org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter] Finished executing task ':mlDeployApp'
13:42:00.800 [LIFECYCLE] [class org.gradle.TaskExecutionLogger] :mlDeployApp FAILED
13:42:00.816 [INFO] [org.gradle.execution.taskgraph.AbstractTaskPlanExecutor] :mlDeployApp (Thread[main,5,main]) completed. Took 17.04 secs.
13:42:00.816 [DEBUG] [org.gradle.execution.taskgraph.AbstractTaskPlanExecutor] Task worker [Thread[main,5,main]] finished, busy: 17.118 secs, idle: 0.0 secs
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter]
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter] FAILURE: Build failed with an exception.
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter]
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter] * What went wrong:
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter] Execution failed for task ':mlDeployApp'.
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter] > Unable to insert content at URI: /ext/corb/process.xqy; cause: Need privilege
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter]
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter] * Try:
13:42:00.816 [ERROR] [org.gradle.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.
13:42:00.816 [LIFECYCLE] [org.gradle.BuildResultLogger]
13:42:00.816 [LIFECYCLE] [org.gradle.BuildResultLogger] BUILD FAILED
13:42:00.816 [LIFECYCLE] [org.gradle.BuildResultLogger]
13:42:00.816 [LIFECYCLE] [org.gradle.BuildResultLogger] Total time: 27.083 secs
13:42:00.831 [DEBUG] [org.gradle.cache.internal.DefaultCacheAccess] Cache task history cache (E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\.gradle\2.12\taskArtifacts) was closed 0 times.
13:42:00.847 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on cp_proj class cache for build file 'E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\build.gradle' (C:\Users\Sam\.gradle\caches\2.12\scripts\build_b0a1q6p13n5dqo2n98dhbvce3\cp_proj).
13:42:00.847 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on proj class cache for build file 'E:\installs\marklogic\marklogic-spring-batch-0.2.4-alpha\build.gradle' (C:\Users\Sam\.gradle\caches\2.12\scripts\build_b0a1q6p13n5dqo2n98dhbvce3\proj).
13:42:00.847 [DEBUG] [org.gradle.cache.internal.btree.BTreePersistentIndexedCache] Closing cache module-metadata.bin (C:\Users\Sam\.gradle\caches\modules-2\metadata-2.16\module-metadata.bin)
13:42:00.847 [DEBUG] [org.gradle.cache.internal.btree.BTreePersistentIndexedCache] Closing cache artifact-at-repository.bin (C:\Users\Sam\.gradle\caches\modules-2\metadata-2.16\artifact-at-repository.bin)
13:42:00.847 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on artifact cache (C:\Users\Sam\.gradle\caches\modules-2).
13:42:00.863 [DEBUG] [org.gradle.api.internal.artifacts.ivyservice.ivyresolve.memcache.InMemoryCachedRepositoryFactory] In-memory dependency metadata cache closed. Repos cached: 1, cache instances: 1, modules served from cache: 0, artifacts: 0
13:42:00.863 [DEBUG] [org.gradle.api.internal.artifacts.ivyservice.resolveengine.store.CachedStoreFactory] Resolution result cache closed. Cache reads: 0, disk reads: 0 (avg: 0.0 secs, total: 0.0 secs)
13:42:00.863 [DEBUG] [org.gradle.api.internal.artifacts.ivyservice.resolveengine.store.CachedStoreFactory] Resolution result cache closed. Cache reads: 0, disk reads: 0 (avg: 0.0 secs, total: 0.0 secs)
13:42:00.863 [DEBUG] [org.gradle.api.internal.artifacts.ivyservice.resolveengine.store.ResolutionResultsStoreFactory] Deleted 2 resolution results binary files in 0.0 secs
13:42:00.863 [DEBUG] [org.gradle.cache.internal.btree.BTreePersistentIndexedCache] Closing cache plugin-use-metadata.bin (C:\Users\Sam\.gradle\caches\2.12\plugin-resolution\plugin-use-metadata.bin)
13:42:00.863 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on Plugin Resolution Cache (C:\Users\Sam\.gradle\caches\2.12\plugin-resolution).
13:42:00.878 [DEBUG] [org.gradle.api.internal.tasks.compile.daemon.CompilerDaemonManager] Stopping 0 compiler daemon(s).
13:42:00.878 [INFO] [org.gradle.api.internal.tasks.compile.daemon.CompilerDaemonManager] Stopped 0 compiler daemon(s).
```

**Comment by sastafford** [comment]
Which version of MarkLogic are you using?  I've tested on 8.0-4 and a clean install of 8.0-5.2 with success

**Comment by sastafford** [comment]
OBE, fixed in #81
